src/core.js

```diff
@@ -228,6 +228,7 @@ define([
             this.opts().core.dataSourcesEndpoint,
             this.opts().core.invalidCaseProperties
         );
+        this.data.core.showOnlyCurrentLang = this.opts().core.showOnlyCurrentLang;
```

Comment:
This should be a javaRosa option

https://github.com/dimagi/Vellum/blob/34a8db231ae022b476126c494dcc786dc0af7324/tests/options.js#L176-L179

not a core option. Although I would not add a value in `tests/options.js` because that will change the default rendering for tests.

Also, this should be moved into

https://github.com/dimagi/Vellum/blob/34a8db231ae022b476126c494dcc786dc0af7324/src/javaRosa/plugin.js#L62-L66

The value should be stored in `this.data.javaRosa` instead of `this.data.core`.

Reply:
Sure @millerdev. I was not sure where to put this. Will move it to `javaRosa` now.